lib/developer_portal/app/views/developer_portal/applications/show.html.liquid

```diff
@@ -190,19 +207,6 @@
               </table>
             </dd>
           </dl>
-          {% if application.can.add_referrer_filters? %}
```

Comment:
The problem with this code (why it was not working on the first page load) is that it references `div#referrer_filters` while the element with this ID is actually a `dl`
```
<dl class="dl-horizontal" id="referrer_filters">
```

So, it could have been fixed by just fixing the selector, but I think adding these callbacks is ugly, and cause unnecessary animation (first rendering the elements, and then fading some of them out), so I applied the same approach as for the application keys - just add `style="display: none;"` conditionally.